src/renderer/components/NewProjectModal/NewProjectView.tsx

```diff
@@ -0,0 +1,127 @@
+import {
+  styled,
+  Stack,
+  Box,
+  TextField,
+  Typography,
+  Button,
+} from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import IconButton from '@mui/material/IconButton';
+import { ChangeEvent, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { makeProjectWithoutMedia } from 'renderer/util';
+import { projectCreated } from 'renderer/store/actions';
+import colors from '../../colors';
+import { Project } from '../../../sharedTypes';
+
+interface Props {
+  closeModal: () => void;
+  nextView: () => void;
+}
+
+const CustomStack = styled(Stack)`
+  width: 100%;
+  height: 100%;
+`;
+
+const CustomColumnStack = styled(CustomStack)`
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-between;
+`;
+
+const CustomRowStack = styled(CustomStack)`
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const Container = styled(Box)`
+  background-color: ${colors.grey[700]};
+  height: 200px;
+`;
+
+const CustomButton = styled(Button)`
+  filter: drop-shadow(0px 8px 16px rgba(0, 0, 0, 0.8));
+`;
+
+const NewProjectView = ({ closeModal, nextView }: Props) => {
+  const [projName, setProjName] = useState<string>('');
```

Comment:
In general it's a good idea to avoid abbreviations in variable names unless they're absolutely necessary - this in itself is fine but it's a slippery slope to obtuse names like 'genImpExpStats' that become meaningless (trust me I've seen it happen 😅)

Reply:
Ah yes, this is something i overlooked before pushing, there is no need for an abbreviated name here. I refactored the name here: https://github.com/patrickbrett/mlvet/pull/38/commits/5545d5ec5937464dc3b775d37bfe19d4e5b354a6

Although its fine to have an abbreviated variable name, changing this just so others dont see an abbreviated variable name and start getting ideas.